openwisp_users/tests/test_admin.py

```diff
@@ -31,7 +31,15 @@ class TestUsersAdmin(TestOrganizationMixin, TestUserAdditionalFieldsMixin, TestC
 
     app_label = 'openwisp_users'
 
-    def _get_edit_form_inline_params(self, user, organization):
+    def _get_org_edit_form_inline_params(self, user, organization):
+        """
+        This function is created to be overridden
+        when the user extends openwisp-users
+        and adds inline forms in the Organization model.
+        """
+        return dict()
+
+    def _get_user_edit_form_inline_params(self, user, organization):
         """
         This function is created to be overridden
         when the user extends openwisp-users
```

Comment:
When working on an app that extends `openwisp_users` the organization and user model's inline forms in the extended apps need to be added in the supplied dictonary to run the tests.

This PR:
1. Improves this features and adds it for organization as well.
2. Adds inline forms in `sample_users` to ensure failure of the same is captured.